src/patient-registration/patient-registration.component.tsx

```diff
@@ -72,7 +72,7 @@ export const initialFormValues: FormValues = {
   deathCause: '',
 };
 
-export const getDeathInfo = (values: FormValues) => {
+export const getDeathInfo = values => {
```

Comment:
We removed the type (`FormValues`) for the input parameter due to the following reason:  In order to write more unit tests instead of component tests (simulating the user input) we wrote two tests for the `getDeathInfo` helper function we introduced.

In the test we at first used an object of type `FormValues` as input to `getDeathInfo`, in which we set a property `deathDate` of type `Date`. The helper method hence obviously returned the `deathInfo` object containing a `deathDate` of type `Date`. We expect it to be of type `string` since this is what we received in the other component tests. We assume, that in the component Formik takes care of the conversion from `Date` to `string`?

In the test (see line 36) we actually would like to provide the following input, so it would match the type `FormValues` and we could bring back the type for the input parameter (which would guarantee that the `isDead` field exists on the input object): 
` const values = { ...initialFormValues, isDead: true, deathDate: new Date('2020-01-01'), deathCause: 'stroke' }; `